### DISCUSSION TITLE
Transcription vs Translation

### DISCUSSION THREAD

**Discussion by 90debasishb** [openai/whisper#1972] [Q&A]
Hi Guys,

I am trying to do transcription from chinese to chinese and translation from chinese to english. One problem i am facing the time point. how do i get the same time point for chinese to english translation as i get when i do the transcription from chinese to chinese. can someone help me in this regard.

**Comment by EtienneAb3d** [comment]
@90debasishb 
You may try to translate using ChatGPT complex processing capabilities. See example here:
https://github.com/EtienneAb3d/ChatMate
